wsd/AdminModel.cpp

```diff
@@ -1159,6 +1159,25 @@ void AdminModel::getMetrics(std::ostringstream &oss)
     oss << "error_unauthorized_request " << UnauthorizedRequestException::count << "\n";
     oss << "error_service_unavailable " << ServiceUnavailableException::count << "\n";
     oss << "error_parse_error " << ParseError::count << "\n";
+    oss << std::endl;
+
+    int tick_per_sec = sysconf(_SC_CLK_TCK);
+    // dump document data
+    for (const auto& it : _documents)
```

Comment:
Whats about the expire state? (should be skipped or be there as label

Reply:
well - expired documents get cleaned up - will leave them in the list for now.